modules/policy/pages/index.adoc

```diff
@@ -338,11 +352,61 @@ For example, the following CR allows you to expose an `nginx` deployment for all
 apiVersion: skupper.io/v1alpha1
 kind: SkupperClusterPolicy
 metadata:
-  name: allowedExposedResources
+  name: allowedexposedresources
 spec:
   namespaces:
     - "*"
   allowedExposedResources: ['deployment/nginx']
 ----
 
 NOTE: For `allowedExposedResources`, each entry must conform to the `type/name` syntax.
+
+// Type: procedure
+[id="exploring-policies"] 
+== Exploring the current policies for a cluster
+
+As a developer you might want to check which policies are enforced for a particular site.
+
+.Procedure
+
+. Log into a namespace where a {skupper-name} site has been initialized.
+
+. Check whether incoming links are permitted:
++
+[source,bash]
+----
+$ kubectl exec deploy/skupper-service-controller -- get policies incominglink
+
+ALLOWED POLICY ENABLED ERROR                                                   ALLOWED BY 
+false   true           Policy validation error: incoming links are not allowed 
+----
++
+In this example incoming links are not allowed by policy.
+
+. Explore other policies:
++
+--
+[source,bash]
+----
+$ kubectl exec deploy/skupper-service-controller -- get policies 
+Validates existing policies
+
+Usage:
+  get policies [command]
+
+Available Commands:
+  expose       Validates if the given resource can be exposed
+  incominglink Validates if incoming links can be created
+  outgoinglink Validates if an outgoing link to the given hostname is allowed
+  service      Validates if service can be created or imported
+----
+
+As shown, there are commands to check each policy type by specifying what you want to do, for example, to check if you can expose an nginx deployment:
+
+[source,bash]
+----
+$ kubectl  exec deploy/skupper-service-controller -- get policies expose deployment nginx
+ALLOWED POLICY ENABLED ERROR                                                       ALLOWED BY 
```

Comment:
It would be nice to include an example showing the `get policies` result for an allowed resource as well.